doc/website/docusaurus.config.js

```diff
@@ -19,68 +19,44 @@ module.exports = {
     navbar: {
       title: 'Atlas',
       logo: {
-        alt: 'My Site Logo',
-        src: 'img/square.svg',
+        alt: 'Atlas',
+        src: 'https://atlasgo.io/uploads/landing/logo.svg',
       },
       items: [
-        // {
-        //   type: 'doc',
-        //   docId: 'getting-started',
-        //   position: 'left',
-        //   label: 'Welcome',
-        // },
-        // {to: '/blog', label: 'Blog', position: 'left'},
         {
           href: 'https://github.com/ariga/atlas',
           label: 'GitHub',
           position: 'right',
         },
       ],
     },
-    footer: {
-      style: 'dark',
-      links: [
+    //
+    "footer": {
+      "links": [
         {
-          title: 'Docs',
-          items: [
-            {
-              label: 'Welcome',
-              to: '/',
-            },
-          ],
+          "title": "Docs",
+          "items": [
+            {"label": "Getting Started", "to": "cli/getting-started/setting-up"},
+            {"label": "Data Definition Language ", "to": "ddl/intro"},
+            {"label": "CLI Reference", "to": "CLI/atlas"},
```

Comment:
```suggestion
            {"label": "CLI Reference", "to": "cli/atlas"},
```

why the URL is uppercased?